example/BUILD.bazel

```diff
@@ -25,6 +26,46 @@ node_package(
 ###########################
 # Fixtures for tests in this file
 
+# Generated bin test as a build action
+rollup(
+    name = "rollup_help",
+    args = ["--help"],
+    # do not call node_package macro for rollup as we did it earlier in the BUILD file.
+    node_package = False,
```

Comment:
hmm. this seems like a smell here to stamp out a node_package or not

Reply:
yeah; looking closer we can drop that. users will have to stamp out the node_package with node_modules() typically and you should be able to use `rollup` here in any package so stamping everywhere wouldn't work anyway